tests/suite/errors/zat.go

```diff
@@ -0,0 +1,17 @@
+package errors
+
+import (
+	"github.com/mccanne/zq/pkg/test"
+)
+
+var Exec = test.Exec{
+	Name:     "error upstream",
+	Command:  `zat simple.zng | zq -`,
+	Input:    test.Trim(input),
+	Expected: "",
+}
```

Comment:
This is really a test of how zq handles empty input, and I think you should dress it as such.

It's also a test we should do for every "zq -f" format, but I don't think that needs to hold up this PR.

Reply:
done